src/domain/transfer/dto.js

```diff
@@ -63,6 +63,18 @@ const fulfilErrorMessageDto = ({ headers, dataUri, params, isFx, logPrefix = ''
   return baseFulfillMessageDto({ action, headers, dataUri, params, logPrefix })
 }
 
+const getMessageDto = ({ headers, params, isFx, logPrefix = '' }) => {
+  const action = isFx ? Action.FX_GET : Action.GET
+  const to = headers[Enum.Http.Headers.FSPIOP.DESTINATION]
+  const from = headers[Enum.Http.Headers.FSPIOP.SOURCE]
+
+  const metadata = makeMessageMetadata(params.id, Type.GET, action)
+  const messageProtocol = StreamingProtocol.createMessageFromRequest(params.id, { headers, dataUri: undefined, params }, to, from, metadata)
+  logger.debug(`${logPrefix}::messageProtocol`, messageProtocol)
```

Comment:
I don't know how this logger got checked in but I believe this re-introduces the issue of string rendering no matter the log level. 

The previous code of `logger.isDebugEnabled && logger.debug()` stopped any string rendering but the enabled check has been moved to after the string is rendered.

You don't have to fix this in your PR.

Reply:
I suppose the string rendering (before the `logger.isDebugEnabled` check) problem happens only when we us `JSON.stringify(...)` on the object parameter before calling the log function. In this case, we're not rendering the object to string before calling the log function.

There are instances in the code where this is not the case and would need to be fixed in a separate PR as suggested for performance reasons.